edge-hub/core/test/Microsoft.Azure.Devices.Edge.Hub.Core.Test/twin/ReportedPropertiesValidatorTest.cs

```diff
@@ -121,6 +121,65 @@ public static IEnumerable<object[]> GetTwinCollections()
                 "Nested depth of twin property exceeds 10"
             };
 
+            yield return new object[]
+            {
+                new TwinCollection(JsonConvert.SerializeObject(new
+                    {
+                        ok = "ok",
+                        complex = new
+                        {
+                            array1 = new[]
+                                {
+                                    new[]
+                                    {
+                                        new[]
+                                        {
+                                            new[]
+                                            {
+                                                new[]
+                                                {
+                                                    new[]
+                                                    {
+                                                        new[]
+                                                        {
+                                                            new[]
+                                                            {
+                                                                new[]
+                                                                {
+                                                                    new[]
+                                                                    {
+                                                                        new[]
+                                                                        {
+                                                                            new[]
+                                                                            {
+                                                                                new[]
+                                                                                {
+                                                                                    new[]
+                                                                                    {
+                                                                                        new[]
+                                                                                        {
+                                                                                            new[] { "one", "two", "three" },
```

Comment:
nit: There may not be a nice way to one line this, so as a little helpful note could we put a comment the new[] that says something like depth 1 or depth 2 or etc... feel free to ignore if feel it is good :)